### ISSUE TITLE
Add COPR build functionality

### ISSUE THREAD

**Issue by phracek** [phracek/rebase-helper#168]
Nowadays we are able to build packages with rpmbuild, mock, fedpkg.
The last one is able to provide a scratch build. But scratch build is available only for 
14 days. 
It would be good to have a build for longer time.
For this propose it would be good to implement support for COPR.

**Comment by phracek** [comment]
@msuchy Would you be able to provide us some know here? Link or something like that?
I guess, we can meet next week.

**Comment by phracek** [comment]
Only rawhide. 
API reference: https://copr.fedorainfracloud.org/api/

**Comment by phracek** [comment]
Command line commands are:
~~~
copr-cli create --chroot fedora-rawhide-x86_64 package_name
copr-cli build phracek/package_name ~/rpmbuild/SRPMS/pycharm-community-2016.1.1-1.fc23.src.rpm  --nowait # it has to initiate fedmsg message
copr-cli status <build_id>
copr-cli download-build <build_id>
~~~

**Comment by phracek** [comment]
@nforro Can you please look on it? Contant person is Mirek Suchy. COPR Lead.

**Comment by nforro** [comment]
Will do.

**Comment by nforro** [comment]
Resolved in pull request #171.
